src/common/notification-creator.ts

```diff
@@ -19,12 +19,14 @@ export class NotificationCreator {
     public createNotification(message: string): void {
         if (message) {
             const manifest = this.browserAdapter.getManifest();
-            this.browserAdapter.createNotification({
-                type: 'basic',
-                message: message,
-                title: manifest.name,
-                iconUrl: '../' + manifest.icons[128],
-            });
+            this.browserAdapter
+                .createNotification({
+                    type: 'basic',
+                    message: message,
+                    title: manifest.name,
+                    iconUrl: '../' + manifest.icons[128],
+                })
+                .catch(console.error);
```

Comment:
Should this be using `createDefaultLogger().error` rather than `console.error` directly?

Reply:
updated